content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -275,33 +275,79 @@ node {
 }
 ----
 
-=== Defining steps
 
+=== Defining global variables
+
+Internally, scripts in the `vars` directory are instantiated as a singleton
+on-demand, when used first.  So it is possible to define more methods,
```

Comment:
s/instantiated as a singleton on-demand, when used first./instantiated as a singleton.  They created on-demand the first time they are called./

s/So it is possible/This makes it possible/